templates/inspec/doc-template.md.erb

```diff
@@ -0,0 +1,19 @@
+<%= lines(autogen_notice :markdown) -%>
```

Comment:
We probably don't want an autogen notice on documentation. You can use the `autogen_exception` function to stop the compiler from yelling at you.